broker/src/main/java/io/camunda/zeebe/broker/bootstrap/JobStreamServiceStep.java

```diff
@@ -96,11 +99,12 @@ public String getName() {
   }
 
   // TODO: replace with real activation properties
-  private record DummyActivationProperties(
+  @VisibleForTesting
+  public record DummyActivationProperties(
```

Comment:
Had to make it public for testing, but this will anyway be removed once we have actual usable implementations from the ZPA team.